README.md

```diff
@@ -30,3 +31,57 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+## Analysis
+
+if statement -> constant... + 1
+
+else statement ->
+
+-for loops $n * n * n * n * n = n^5$
+
+-recursion $3T(n/3)$ -> 3 recursion calls with n/3 each time
+
+=> $3T(n/3)+n^5$
+
+plug it into itself
+
+=> $3(3T(n/3/3)+(n/3)^5) + n^5$     =>     $9T(n/9)+3(n/3)^5 + n^5$
+
+one more time
+
+=> $27T(n/27) + 9(n/9)^5 + 3(n/3)^5 + n^5$ 
+
+
+=> $3^iT(n/3^i) + n^5 \sum_{j=0}^{i-1} (1/3^{4(j-1)})$
```

Comment:
Still don't see how you would get the factor of e.g. 9.

Reply:
I changed the bounds of the sum to be $\sum_{j=1}^{i}$

here is my work for the j = 3 iteration of the sum to show how to get from $n^5(\frac{1}{3^{4(2)}})$ to $9(\frac{n}{9})^5$: 

$n^5(\frac{1}{3^{4(2)}})$ 

=> $\frac{n^5}{3^{4(2)}}$ 

=> $\frac{n^5}{9^{4}}$ 

=> $\frac{n^5}{9^{4}}(\frac{9}{9})$ 

=> $\frac{9n^5}{9^5}$ 

=> $9(\frac{n}{9})^5$